Gems/XR/Code/Include/XR/XRInput.h

```diff
@@ -27,21 +27,16 @@ namespace XR
         AZ_CLASS_ALLOCATOR(Input, AZ::SystemAllocator, 0);
         AZ_RTTI(Input, "{DCDFC6A7-B457-414B-BC24-0831C2AC628B}");
 
-        Input() = default;
-        virtual ~Input() = default;
-        
```

Comment:
Why removing the virtual destructor?

Reply:
Base `XR::Object` already defines a (default) virtual destructor, so these aren't needed as they aren't doing anything special.

Reply:
Good call! Thanks